### PR TITLE
T125747: Use _stopWorker in master shutdown

### PR DESCRIPTION
**Pull request by arlolra**

### CODE REVIEW

File: lib/master.js [hunk 1]
```diff
@@ -65,7 +65,10 @@ Master.prototype._run = function() {
         if (self.interval) {
             clearInterval(self.interval);
         }
-        cluster.disconnect(function() {
+        P.each(Object.keys(cluster.workers), function(workerId) {
```
Comment (gwicke) [inline lib/master.js]: It might be okay to stop all at once, with P.all / map.

**Comment by gwicke** [comment]
:+1: 

/cc @wikimedia/services

**Comment by arlolra** [comment]
`P.map(Object.keys(cluster.workers), self._stopWorker)` would have been sufficient :(

**Comment by d00rman** [comment]
Indeed @arlolra . Weird we we haven't spotted it sooner.

**Comment by arlolra** [comment]
I pushed 2012443f7c214288a1c3b6bc6a8f0a1cabfe39d4.

**Pull request merged**